CMakeLists.txt

```
@@ -93,8 +99,14 @@ endif ()
 
 set (OPENCL_ICD_LOADER_HEADERS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/inc" CACHE PATH "Path to OpenCL Headers")
 
-add_library (OpenCL ${OPENCL_ICD_LOADER_SOURCES})
+if (${OPENCL_ICD_LOADER_BUILD_SHARED_LIBS} OR ${BUILD_SHARED_LIBS})
+  add_library (OpenCL SHARED ${OPENCL_ICD_LOADER_SOURCES})
+else()
+  add_library (OpenCL SHARED ${OPENCL_ICD_LOADER_SOURCES})
```

This looks like a typo because it matches the "if" block above.  Should this be:

```suggestion
  add_library (OpenCL STATIC ${OPENCL_ICD_LOADER_SOURCES})
```